SPECS/kpatch/kpatch.spec

```diff
@@ -72,6 +78,12 @@ rm -rf %{buildroot}%{_mandir}
 %{_bindir}/kpatch-build
 
 %changelog
+* Mon Aug 01 2022 Pawel Winogrodzki <pawelwi@microsoft.com> - 0.9.6-4
+- Adding missing dependency on "elfutils" and "awk".
```

Comment:
Looks like the changelog lists `elfutils`, but the actual change says `binutils`.